components/BridgeProgress.tsx

```diff
@@ -136,7 +136,7 @@ const TxFailure: VFC<IntrinsicElements["div"] & TxFailureProps> = ({
 			<h1>Transaction Failed</h1>
 			<p>
 				An error occurred while processing your transaction. It might have gone
-				through, check your balances before trying again.
+				through, check your balances before try again.
 			</p>
```

Comment:
this was fine as is, or you could say `check your balances before you try again`